### PR TITLE
Re code by @LuxSyphurithSIe

### PR DESCRIPTION
**Pull request by AlbertoSONIC**

### CODE REVIEW

**Comment by Syphurith** [comment]
Well so glad to see all these done. Now `commented` can be added to the `README.md` if you pleased.

File: CFW_loader/source/main.c [hunk 1]
```diff
@@ -7,226 +7,191 @@
...
-	case 'a': // 8.x
-		type = 'a';
-		systemVersion = "New 3DS V. 8.1";
-		break;
-	case 'b': // 9.x
-		type = 'b';
-		systemVersion = "New 3DS V. 9.0 - 9.2";
-		break;
+    cfw_FWValue = settings[0];
+	switch (settings[0]) {
```
Comment (Syphurith) [inline CFW_loader/source/main.c:38]: If we adjust how the `cfw_FWValue` store its value, this convertion would be done much easier. _Example_ `'1'` to binary `0x01` or `'1'` to `'a'`.and `systemVersion=cfw_FWString[cfw_FWValue-1]`.
Also a marker telling which version the configuration file may be introduced.
To do such jobs, a normal configuration file struct would be discussed.

**Comment by Syphurith** [comment]
@AlbertoSONIC lol the @LuxSyphurithSIe is indeed myself. The email address mismatch my github profile, so it created a new user unable to be logged-in and prevents me from changing the email...
I'm now thinking of a standard of the CFW Options.. and porting @felipejfc 's YGW.. and Custom-Patch.

**Comment by felipejfc** [comment]
@Syphurith porting YGW into the CFW itself, making it auto-load is way trickier than it seems, you'ld need to do some king of code inception from arm9 to arm11... I will be very happy if you manage to do so.

Regards

**Comment by Syphurith** [comment]
@felipejfc Orz. Still reading your code and trying to find the route. You used `ctrcommon` to retrieve the kernel, which is indeed provided by `libkhax`. Well.. I think i am out of some luck currently.

**Pull request merged**